eZ/Bundle/EzPublishIOBundle/BinaryStreamResponse.php

```diff
@@ -133,6 +137,11 @@ public function setContentDisposition( $disposition, $filename = '', $filenameFa
      */
     public function prepare( Request $request )
     {
+        if ( !$this->file instanceof BinaryFile )
```

Comment:
And It's not really the BinaryStreamResponse responsibility to check for this imho, so lets not mix that concern in here.